src/cryptoadvance/specter/wallet.py

```diff
@@ -363,6 +364,28 @@ def rescanutxo(self, explorer=None):
         t = threading.Thread(target=self._rescan_utxo_thread, args=(explorer,))
         t.start()
 
+    def export_labels(self):
+        labels = self.rpc.listlabels()
+        if "" in labels:
+            labels.remove("")
+        res = self.rpc.multi([
+            ("getaddressesbylabel", label)
+            for label in labels
+        ])
+        return [{ labels[i]: list(result['result'].keys()) } for i, result in enumerate(res)]
+    
+    def import_labels(self, labels):
+        # format: [
+        #   {'label': ['address', 'address2']}
+        # ]
```

Comment:
Why not like this?
```
{
 "label": [addresses],
 "another label": [more addresses]
}
```
What's the point of an array?

Reply:
No point, I initially wanted to structure it slightly differently and didn't think about that, fixed.